src/p4pktgen/switch/runtime_CLI.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```

Comment:
I do not know how important this difference is, but in some other p4lang projects there have been suggestions made to use python3 explicitly when updating things for Python 3, perhaps because there is a Python PEP recommending this.  at the very least, it ensures that if someone still has a system where python is Python 2, python3 should be Python 3.

Reply:
I was just leaving this as it is in the p4lang project.  I have no particular preference either way, but thought that it should differ from the base version as little as possible.

Reply:
Understood.